### ISSUE TITLE
%paths.base% not taken into account for screenshot_directory

### ISSUE THREAD

**Issue by leymannx** [elvetemedve/behat-screenshot#46]
Taking the following `behat.yml` the screenshots will be saved into Drupal's root dir (`drupal_root: ../../web`) as directory named `%paths.base%/screenshots/failed_steps`.

So it seems that `screenshot_directory` currently can only be an absolute path or a relative path to the target projects's web root. But it MUST also be possible to use `%paths.base%` to make it relative to the current `behat.yml`.

```yaml
default:
  suites:
    default:
      contexts:
        - FeatureContext
        - Drupal\DrupalExtension\Context\DrupalContext
        - Drupal\DrupalExtension\Context\MinkContext
        - Drupal\DrupalExtension\Context\MessageContext
        - Drupal\DrupalExtension\Context\DrushContext
  extensions:
    Behat\MinkExtension:
      base_url: http://drupal.localhost
      goutte: ~
      selenium2:
        wd_host: http:/drupal.localhost:4444/wd/hub
        capabilities:
          marionette: true
          browser: chrome
      browser_name: chrome
    Drupal\DrupalExtension:
      blackbox: ~
      api_driver: drupal
      drupal:
        drupal_root: %paths.base%/../../web
    Bex\Behat\ScreenshotExtension:
      screenshot_taking_mode: failed_steps
      image_drivers:
        local:
          screenshot_directory: %paths.base%/screenshots/failed_steps
```

**Comment by tkotosz** [comment]
Hi @leymannx 

Thanks for the bug report, I will prepare a fix for this today. :)

**Comment by elvetemedve** [comment]
It's fixed in release [1.2.8](https://github.com/elvetemedve/behat-screenshot/releases/tag/1.2.8).

**Comment by tkotosz** [comment]
@leymannx pls let me know if there is any problem with the fix. :)
